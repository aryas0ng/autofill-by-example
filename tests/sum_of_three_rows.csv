11,28
15,22
2,10
5,10
3,9
2,
4,
43,
5,
6,
9,
3,
4,
2,
5,
2,
6,
1,
8,
6,
1,
21,
3,