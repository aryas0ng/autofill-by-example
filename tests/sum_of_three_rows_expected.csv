11,28
15,22
2,10
5,10
3,9
2,49
4,52
43,54
5,20
6,18
9,16
3,9
4,11
2,9
5,13
2,9
6,15
1,15
8,15
6,28
1,25
21,24
3,3